examples/logs-agent/main.tf

```diff
@@ -0,0 +1,178 @@
+  create_duration = "5s"
+}
+
+##############################################################################
+# Observability Instance
+##############################################################################
+
+
+module "observability_instance" {
+  source  = "terraform-ibm-modules/observability-instances/ibm"
+  version = "2.16.0"
+  providers = {
+    logdna.at = logdna.at
+    logdna.ld = logdna.ld
+  }
+  region                     = var.region
+  log_analysis_provision     = false
+  cloud_monitoring_provision = false
+  activity_tracker_provision = false
+  cloud_logs_instance_name   = "${var.prefix}-cloud-logs"
+  resource_group_id          = module.resource_group.resource_group_id
+  cloud_logs_plan            = "standard"
+  cloud_logs_tags            = var.resource_tags
+  cloud_logs_access_tags     = var.access_tags
+}
+
+data "ibm_is_security_groups" "vpc_security_groups" {
+  depends_on = [ibm_container_vpc_cluster.cluster]
+  vpc_id     = ibm_is_vpc.example_vpc.id
+}
+
+module "vpe" {
+  source   = "terraform-ibm-modules/vpe-gateway/ibm"
+  version  = "4.3.0"
+  region   = var.region
+  prefix   = var.prefix
+  vpc_id   = ibm_is_vpc.example_vpc.id
+  vpc_name = "${var.prefix}-vpc"
+  subnet_zone_list = [
+    {
+      name = ibm_is_subnet.testacc_subnet.name
+      id   = ibm_is_subnet.testacc_subnet.id
+      zone = ibm_is_subnet.testacc_subnet.zone
+    }
+  ]
+  resource_group_id  = module.resource_group.resource_group_id
+  security_group_ids = [for group in data.ibm_is_security_groups.vpc_security_groups.security_groups : group.id if group.name == "kube-${ibm_container_vpc_cluster.cluster.id}"] # Select only security group attached to the Cluster
+  cloud_service_by_crn = [
+    {
+      crn          = module.observability_instance.cloud_logs_crn
+      service_name = "logs"
+    }
+  ]
+  service_endpoints = "private"
+}
+
+##############################################################################
+# Observability Agents
+##############################################################################
+
+module "observability_agents" {
+  source                                 = "../../modules/logs-agent-module"
+  depends_on                             = [time_sleep.wait_operators, module.vpe]
+  cluster_id                             = ibm_container_vpc_cluster.cluster.id
+  cluster_resource_group_id              = module.resource_group.resource_group_id
+  logs_agent_enabled                     = true
+  logs_agent_trusted_profile             = module.trusted_profile.trusted_profile.id
+  logs_agent_namespace                   = local.logs_agent_namespace
+  logs_agent_name                        = local.logs_agent_name
+  logs_agent_enable_direct_to_cloud_logs = true
+  cloud_logs_ingress_endpoint            = module.observability_instance.cloud_logs_ingress_private_endpoint
+  cloud_logs_ingress_port                = 443
+  logs_agent_enable_scc                  = var.is_openshift ? true : false
```

Comment:
Can we also add some mock values for `logs_agent_additional_metadata` and `logs_agent_additional_log_source_paths` so that we are at least testing that functionality